package.json

```diff
@@ -45,7 +45,7 @@
     "@babel/runtime": "^7.14.0",
     "@headlessui/react": "^1.6.0",
     "@heroicons/react": "^1.0.5",
-    "@pollum-io/sysweb3-core": "^1.0.21",
+    "@pollum-io/sysweb3-core": "1.0.19",
```

Comment:
If possible change sysweb3-core version

Reply:
It's not possible, the new version is not working for some functionalities